packages/slice-machine/src/modules/selectedSlice/sagas.ts

```diff
@@ -33,7 +33,7 @@ import Tracker from "../../tracking/client";
 export function* generateSliceScreenshotSaga({
```

Comment:
@Heyrows I think this is what you were saying should be extracted into a separate module correct? Do you think it's own GenerateSliceScreenshot module for the saga and actions is good?

Reply:
Today it is entirely linked to the Slice Builder page as it was the only place to take a screenshot.
Now, we'll be able to
- take a screenshot from the simulator
- use a custom screenshot from every screens

The simulator has its own module & store so it would make sense to put the `RegularScreenshotSaga` there as well.
However, not keeping the screenshots saga together would be wrong i think.

I'm no Redux expert so I don't know which code structure people usually use, what do you think @danigo99 ?

Reply:
@Heyrows Is the custom-screenshot feature also going to move to the simulator page? I think I would create a separate module (probably only action and saga since there's no state) to handle it, and the events could be called from both anywhere where it makes sense to upload a screenshot.
What do you think?

Reply:
Hey, the custom screenshots feature is going to be in a modal that will be open all across the app but not in the Simulator page.

Agree those two sagas are going to be independent from any modules 😄.
It is like the `pushChangesSaga`